packages/calypso-data-stores/src/domain-suggestions/selectors.ts

```diff
@@ -16,7 +16,7 @@ export const getState = ( state: State ) => state;
 type DomainSuggestionSelectorOptions = Partial< Exclude< DomainSuggestionQuery, 'query' > >;
 
 export const getDomainSuggestions = (
-	state: State,
+	_state: State,
```

Comment:
Unused in the function body. The leading underscore makes TypeScript ignore it (it'd otherwise complain about the unused arg).

Reply:
Nice convention 🙂 I think Rust does something similar, too.